### ISSUE TITLE
Web accessibility issues

### ISSUE THREAD

**Issue by chunkitdesign** [ExpressionEngine/ExpressionEngine#1590]
**Is your feature request related to a problem? Please describe.**
I ran the AXE accessibility test on the EE6 CP and it lists more than 90 accessibility issues. around 26 issues are on the colour contrasts. For example, the instruction text under the fields is only 12px and the colour contrast (#8F90B0) does not pass the test—the contrast rate is only 3.09. With such a low colour contrast with small types, it is hard for me to convince my clients to switch to this new UI.
**There are just too many issues.**
[screenshot](https://www.dropbox.com/s/0khfxcms674ihsz/CleanShot%202021-11-28%20at%2014.22.29%402x.png?dl=0)

**Describe the solution you'd like**

1. Please increase the font size
2. make the contrast pass the test
3. Please run the Wave or [AXE Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) by deque 

**Describe alternatives you've considered**
n/a

**Teachability, Documentation, Adoption, Migration Strategy**

- [ ] I am capable and would like to work on the implementation of this feature if it is considered.

**Comment by murtaugh** [comment]
Accessibility isn't a nice to have, it's critical for supporting as many users as possible, not to mention critical to allowing EE to be used for projects for public/governmental clients.

**Comment by chunkitdesign** [comment]
Totally agree with you. And I hope that the custom CSS in the CP will be available for the free version, not the Pro version only because I need to make the color contrast good for my client who is of the higher education sector in Ontario.

**Comment by blue-dreamer** [comment]
I agree as well. Fixing colour contrasts should be fairly straightforward.
